addons/stock/migrations/13.0.1.1/post-migration.py

```diff
@@ -344,13 +344,7 @@ def map_stock_locations(env, main_company):
     env["stock.quant.package"].search([])._compute_package_info()
 
     # xmlids are deprecated in v13
```

Comment:
Delete these lines directly without making any comment. You have work file for indicating that.

Reply:
done